pkg/yurtctl/cmd/convert/edgenode.go

```diff
@@ -155,6 +141,12 @@ func (c *ConvertEdgeNodeOptions) Complete(flags *pflag.FlagSet) error {
 	}
 	c.KubeadmConfPath = kubeadmConfPath
 
+	podManifestPath, err := enutil.GetPodManifestPath(c.KubeadmConfPath)
+	if err != nil {
+		return err
+	}
+	c.PodMainfestPath = podManifestPath
```

Comment:
As `podManifestPath` only use in func `SetupYurthub()`, how about remove `c.PodManifestPath`?

Reply:
updated